mne/io/egi/egi.py

```diff
@@ -192,22 +189,16 @@ def __init__(self, input_fname, montage=None, eog=None, misc=None,
             logger.info('Reading EGI header from %s...' % input_fname)
             egi_info = _read_header(fid)
             logger.info('    Reading events ...')
-            _read_events(fid, egi_info)  # update info + jump
```

Comment:
i.e. here, output never used. And the forwarding of the file pointer was never made use of, either, since the next reading function, the old `_read_data` used a call to `fid.seek(info['bytes_per_slice'] * start, 0)` which is an absolute seek.